### PR TITLE
Add missing declaration files

### PR DESCRIPTION
**Pull request by hujiulong**

### CODE REVIEW

File: src/renderers/webvr/WebXRManager.d.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+	enabled: boolean;
+	getController( id: number ): Group;
+	setFramebufferScaleFactor( value: number ): void;
+	setReferenceSpaceType( value: string ): void;
+	getSession(): any;
+	setSession( value: any ): void;
+	getCamera( camera: Camera ): Camera;
+	isPresenting: () => boolean;
+	setAnimationLoop( callback?: Function ): void;
```
Comment (Mugen87) [inline src/renderers/webvr/WebXRManager.d.ts]: I would say `callback` is a mandatory parameter.

Comment (Mugen87) [reply src/renderers/webvr/WebXRManager.d.ts]: BTW: Can you please add `setAnimationLoop ()`, `isPresenting()` and `getStandingMatrix()` to `WebVRManager.d.ts`? The file seems a bit out of date...

Comment (hujiulong) [reply src/renderers/webvr/WebXRManager.d.ts]: @Mugen87 Done~

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**